<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScientistDialog</class>
 <widget class="QDialog" name="AddScientistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_add_scientist">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#5555ff;&quot;&gt;Add a new scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_error_scientist_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_scientist_name">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_Female">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>female</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_Male">
     <property name="text">
      <string>male</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_error_scientist_year_born">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_scientist_year_born">
     <property name="placeholderText">
      <string>Year born</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error_scientist_year_death">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ScientistIsDead">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>The scientist is dead</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="input_scientist_year_death">
        <property name="placeholderText">
         <string>Year of death</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>input_scientist_year_death</zorder>
     <zorder>input_scientist_year_death</zorder>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_add_scientist">
     <property name="text">
      <string>Add scientist</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
